lib/src/blaze/block_witness_data.rs

```diff
@@ -3,10 +3,11 @@ use crate::{
     grpc_connector::GrpcConnector,
     lightclient::checkpoints::get_all_main_checkpoints,
     wallet::{
-        data::{BlockData, WalletTx, WitnessCache},
+        data::{BlockData, FromCommitment, WalletNullifier, WalletTx, WitnessCache},
```

Comment:
`WalletNullifier` is interesting terminology....

Reply:
Keeping WalletNullifier (at least for now) in order to avoid adding separate channels for sapling/orchard nullifiers.